packages/server/medplum.config.json

```diff
@@ -42,5 +42,6 @@
     "removeOnFail": { "count": 1 },
     "removeOnComplete": { "count": 1 }
   },
-  "shutdownTimeoutMilliseconds": 30000
+  "shutdownTimeoutMilliseconds": 30000,
+  "chainedSearchWithReferenceTables": true
```

Comment:
👍  - this makes sense.  we should all make mental note of this if/when developers experience weird behavior on localhost